packages/webapps/lib/hooks/useTeamMembers.ts

```
@@ -0,0 +1,123 @@
+    };
+
+    return member;
+  }, [memberData, memberAddress]);
+}
+
+// Hook to get all members for a team
+// Note: Since we don't have a way to enumerate all members from the contract,
+// this hook expects a list of member addresses to be provided
+export function useTeamMembers(teamId: number, memberAddresses: string[] = []) {
+  const memberResults = memberAddresses.map((address) =>
+    useTeamMember(teamId, address)
```

Biome (2.1.2)

[error] This hook is being called from a nested function, but all hooks must be called unconditionally from the top-level component.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)